WoWsShipBuilder.Desktop/Features/BlazorWebView/DesktopTopMenu.razor

```diff
@@ -21,6 +21,11 @@
     <NavLink href="/charts" class="main-nav-link" ActiveClass="nav-link-active">@Localizer.SimpleAppLocalization(nameof(Translation.NavigationCharts))</NavLink>
     <NavLink href="/acceleration-charts" class="main-nav-link" ActiveClass="nav-link-active">@Localizer.SimpleAppLocalization(nameof(Translation.NavigationAccelerationCharts))</NavLink>
     <NavLink href="/comparison" class="main-nav-link" ActiveClass="nav-link-active">@Localizer.SimpleAppLocalization(nameof(Translation.NavigationComparison))</NavLink>
+    <MudDivider DividerType="DividerType.Middle" FlexItem Vertical/>
+    <MudButton StartIcon="@Icons.Material.Filled.Settings" IconSize="Size.Large" Color="Color.Inherit" OnClick="@OpenSettings" Target="_blank" Size="Size.Medium">
```

Comment:
While a visual separation of the settings button is fine, it should not be fully upper-case if the other nav links are normal text